pinns/src/pinns.c

```diff
@@ -63,28 +66,34 @@ int main(int argc, char **argv) {
   while ((c = getopt_long(argc, argv, "pchuUind:f:s:", long_options, NULL)) != -1) {
     switch (c) {
     case 'u':
-      unshare_flags |= CLONE_NEWUTS;
+      if (is_host_ns (optarg))
```

Comment:
You need to add a `!` for the `is_host_ns` invocations.

Reply:
what a foolish mistake :upside_down_face: thanks, fixed!